pkg/kudoctl/packages/reader/read_dir.go

```diff
@@ -46,6 +47,16 @@ func FromDir(fs afero.Fs, packagePath string) (*packages.Files, error) {
 	if packagePath == "" {
 		return nil, errors.New("path must be specified")
 	}
+
+	if !filepath.IsAbs(packagePath) {
+		// Normalize package path to provide more meaningful error messages
+		absPackagePath, err := filepath.Abs(packagePath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "Failed to normalize package path %s", packagePath)
```

Comment:
Overall looks good! We're reducing our dependency on pkg/errors everywhere. Check out: https://blog.golang.org/go1.13-errors. In short, instead you'll use fmt.Errorf with the new `%w` verb to wrap errors. A good example of this in our codebase is here: https://github.com/kudobuilder/kudo/blob/master/pkg/engine/task/task_apply.go#L105

Reply:
or rather %v if that error is printed out :)

Reply:
Ah, good to know. Replaced all usages of pkg/errors in read_dir.go